spark/dl/src/main/scala/com/intel/analytics/bigdl/dataset/segmentation/MaskUtils.scala

```diff
@@ -77,6 +78,34 @@ class RLEMasks(val counts: Array[Int], val height: Int, val width: Int) extends
   def get(idx: Int): Long = {
     MaskUtils.uint2long(counts(idx))
   }
+
+  override def equals(obj: Any): Boolean = {
+    if (obj == null) {
+      return false
+    }
+    if (!obj.isInstanceOf[RLEMasks]) {
+      return false
+    }
+    val other = obj.asInstanceOf[RLEMasks]
+    if (this.eq(other)) {
+      return true
```

Comment:
is it true？

Reply:
yeah, when they equal, it should be true